### ISSUE TITLE
Store|Woo: Rework "Dashboard" to just have a notice about the move to WooCommerce

### ISSUE THREAD

**Issue by timmyc** [Automattic/wp-calypso#48029]
**Important** Place this behind the "store-post-sunset" feature flag (implementation tbd)

Was:  ~Store|Woo: Replace all `/store/*` routes with one landing page~

_Edit by @becdetat: making this just about the dashboard page, moving route changes to separate issues as this issue's scope had changed a bit over time, and the route changes can be implemented in parallel._ 

> This issue is part of a project to migrate users of the Store on WordPress.com feature ( The Woo UI in calypso ) to use the default WooCommerce experience in wp-admin instead. Further details & discussion: pbIJXs-sE-p2

This part of the project will launch when the sunset date arrives. ~After considering doing something like redirects for each route in calypso -> wp-admin/woo, I'm thinking it might be best to simply retain the Store UI for a bit, with the [notice](https://github.com/Automattic/wp-calypso/issues/48026) being shown about the move to the wp-admin/Woo user interface. But definitely would love some input on this from @pmcpint and @elizaan36.~ 

Roughly here is what I'm thinking we could do:

<img width="1327" alt="store-post-sunset" src="https://user-images.githubusercontent.com/22080/101109108-f3625980-358a-11eb-8c30-0da9073e4e78.png">

This ticket removes most of the existing dashboard and replaces it with a notice that the Store has moved to WooCommerce. Here's a potential design from @elizaan36 :

![](https://user-images.githubusercontent.com/5121465/101561962-f1820700-398b-11eb-9f15-7a627a3f712a.png)

~To implement, we can update [all the Store routes](https://github.com/Automattic/wp-calypso/blob/trunk/client/extensions/woocommerce/index.js#L48-L186) to point at the dashboard page, and on that page we will retain the messaging about the UI being sunset, and also have a link to the relevant support documentation, and a reminder of how to access the new UI using the WooCommerce link in the sidebar.~

~I also think it might be nice to update the links in the Store Sidebar ( Products, Orders, Reviews etc ) to link to their corresponding views in wp-admin.~

**Comment by pmcpinto** [comment]
> I also think it might be nice to update the links in the Store Sidebar ( Products, Orders, Reviews etc ) to link to their corresponding views in wp-admin.

@timmyc but that would mean that they won't be able to access the Store features during the transition period, correct? Or maybe I'm missing something here.

**Comment by timmyc** [comment]
@pmcpinto this piece of the work is what would effectively turn off the Store on WPCOM UI, so we wouldn't push this change live until after the sunset date.

#48026 & #48027 would be how the store UI would look during the sunset period:

- Store menu item which opens Store on WPCOM UI
- WooCommerce menu item which opens in wp-admin
- Banner on dashboard in store UI which informs them of the sunset details

**Comment by pmcpinto** [comment]
> @pmcpinto this piece of the work is what would effectively turn off the Store on WPCOM UI, so we wouldn't push this change live until after the sunset date.

👍 makes sense. Thanks for clarifying this.

**Comment by elizaan36** [comment]
Would stats still appear in the dashboard? If so, to keep it simple we could keep the Dashboard as-is but just redirect the links. The notice would inform users that `Store has moved to WooCommerce`.
![Frame 642](https://user-images.githubusercontent.com/5121465/101401979-ba7ef900-3898-11eb-8c69-26cd6db43e25.png)

For the nav issue, we could present a tooltip on the Store nav item like this, which informs users that `Store is moving to WooCommerce`.
<img width="507" alt="Screen Shot 2020-12-07 at 2 14 57 PM" src="https://user-images.githubusercontent.com/5121465/101402076-e00c0280-3898-11eb-91b3-e5c2fb2fe34f.png">

Both the notice and tooltip message would have a link to the doc.

**Comment by timmyc** [comment]
> Would stats still appear in the dashboard?

I would prefer that we don't use any of the existing components/logic of Store on WPCOM and instead update the dashboard to fully reflect that the new place to manage their store exists in wp-admin. Since there is no "Stats" option in the sidebar, perhaps we could just have a link to the stats overview page in its place.

**Comment by elizaan36** [comment]
Ok, so we need a new card on this screen with the messaging about how Store has been moved, with links out to the right place to get things done/to learn more.

**Comment by elizaan36** [comment]
Here's a potential design for the dashboard landing screen after sunset of Store.

![After sunset](https://user-images.githubusercontent.com/5121465/101561962-f1820700-398b-11eb-9f15-7a627a3f712a.png)

It's showing the full width label with link to documentation. Then the primary action on the page goes to WooCommerce Home.

As Timmy mentioned, all the links in the left nav could redirect to the corresponding areas of WooCommerce.

**Comment by becdetat** [comment]
As the conversation here moved more towards the home page and not about the routes/menu items, I've moved the menu item rerouting into seperate issues and edited the description of this issue to just be about the home page change.

**Comment by moon0326** [comment]
Do we already have a link to documentation by any chance?

**Comment by timmyc** [comment]
> Do we already have a link to documentation by any chance?

@moon0326 I think we'll just use the existing "Store" documentation page and update it with the new content at the start of the sunset period: https://wordpress.com/support/store/

**Comment by moon0326** [comment]
@timmyc Thank you 👍 I will use that link for now.
